Content.Server/GameObjects/Components/Arcade/BlockGameArcadeComponent.cs

```diff
@@ -42,6 +42,17 @@ public class BlockGameArcadeComponent : Component, IActivate
         private IPlayerSession? _player;
         private readonly List<IPlayerSession> _spectators = new();
 
+        public override void HandleMessage(ComponentMessage message, IComponent ?component)
```

Comment:
```suggestion
        public override void HandleMessage(ComponentMessage message, IComponent? component)
```

There's a few of these